### ISSUE TITLE
Allow students to request more courses

### ISSUE THREAD

**Issue by lbkchen** [calblueprint/ros-student#277]
Some details might change based on Chad's decision, but for now, you should make the following:
* Include an extra page on student-side that allows students to request for more courses once they've completed **all of their currently enrolled courses**
* The page should instead display an message like "Please complete your courses to request more."  (pls style this) if they're not done yet
* The page should probably just be a grid of cards but you could play around and make it look slightly different. Basically the functionality is that the student should be able to select any subset of the currently published courses that they are *not already enrolled in* and finally submit their request. They should also be able to choose for each course whether they want to take it self-paced. You have free reign over what this looks like but make sure you include enough info for the student to get a good idea of what the course(s) are
* Change the "Codes" page on admin side to something like "Enrollment" and give it two tabs, one for code generation and one for accepting student course requests
* The accepting student course requests tab should display a list of the students' requests for enrolling in more courses. The request should have the student name, picture, time/date requested, a list of all currently enrolled courses, and a list of all additional courses requested. The admin should either accept the entire request or reject the entire request. If the admin chooses the reject the request however, there should be an input field that allows the admin to include a message with the rejection, which should be emailed to the student.

Due by end of sprint but I might give you another small task 🙉 

Basically Jason should mainly focus on the back end/functionality. Kenneth should mainly focus on the UI/UX, design, and styling of all the pages such that the flow is natural and the styles match our current thene

**Comment by lbkchen** [comment]
finally done!
